tests/functional/t/ternary.py

```diff
@@ -1,18 +1,21 @@
 """Test for old ternary constructs"""
-from UNINFERABLE import condition, true_value, false_value, some_callable  # pylint: disable=import-error
+from UNINFERABLE import condition, some_callable  # pylint: disable=import-error
```

Comment:
The previous tests were wrong IMO, they caused the same false positive as reported in the issue. If something is uninferable, we should not assume it's truthy!

I agree I can make this more explicit with a test.

Reply:
> If something is uninferable, we should not assume it's truthy!

Agree. Maybe we can keep only one test for this scenario instead of what we had before ?